docs/run_simulation/windows.rst

```diff
@@ -0,0 +1,199 @@
+On Windows 
+==========
+
+Prerequisites
++++++++++++++
+
+1. Clone the `GCBM Colombia <https://github.com/moja-global/GCBM.Colombia>`_ repository
+
+2. Download the installation tools from releases: `install_tools <https://github.com/moja-global/GCBM.Colombia/releases/tag/install_tools>`_, and unzip them into `Standalone_GCBM\\tools` folder. On unzipping, `Standalone_GCBM\\tools` will contain a folder tools with the folders `python_3_installer` and `VC_redist` .
+The configuration is **strictly for Python3.7**, it will not work with other versions
+
+
+.. figure:: ../assets/install_tools.PNG
+   :alt: Unzipped contents of install tools
+   :align: center
+   :width: 600px
+
+
+Running the simulation (short version)
+--------------------------------------
+
+Navigate to `Standalone_GCBM`, edit `run_all.bat` and set the `GCBM_PYTHON path` and `PLATFORM` variables in the `USER CONFIGURATION` section near the top to the correct values for your system
+
+Run `run_all.bat`, which performs the following preprocessing, and simulation
+
+Running the project (long version)
+----------------------------------
+
+This involves running each stage of the simulation step by step.
+It helps in understanding the sequence of steps, inputs, outputs and dependencies to each step 
+
+Step 1 
+------
+
+Working directory : **Standalone_GCBM\\layers\\tiled**
+
+..  code-block:: bash
+    :caption: Command
+
+        c:\\python37\\python.exe ..\\..\\tools\\Tiler\\tiler.py
+
+Description :  
```

Comment:
this description is for what ? i think you should write a little note underneath

Reply:
Yes, the description is mentioned as pointer under the heading